File: backend/community/src/main/java/com/avocado/community/common/error/ErrorResp.java
package com.avocado.community.common.error;

import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;
import org.springframework.http.HttpStatus;

@AllArgsConstructor
@NoArgsConstructor
public class ErrorResp {
    public HttpStatus status;
    public String msg;

    public ErrorResp(ResponseCode code) {
        this.status = code.status;
        this.msg = code.msg;
    }

}

File: backend/user-server/src/main/kotlin/com/avocado/userserver/common/error/ErrorResp.kt
package com.avocado.userserver.common.error

import org.springframework.http.HttpStatus

class ErrorResp(status:HttpStatus, msg:String) {
    public val status: HttpStatus = status
    public val msg: String = msg

    constructor(code: ResponseCode): this(code.status, code.msg)
}